scene/resources/packed_scene.cpp

```diff
@@ -320,7 +320,7 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 		// we only want to deal with pinned flag if instancing as pure main (no instance, no inheriting)
 		if (p_edit_state == GEN_EDIT_STATE_MAIN) {
 			_sanitize_node_pinned_properties(node);
-		} else {
+		} else if (node != nullptr) {
```

Comment:
```suggestion
		} else if (node) {
```
That's enough as a null check.